tests/lms/views/error_test.py

```diff
@@ -90,3 +90,11 @@ def test_it_sets_correct_status_int_for_missing_lti_param_error(self, pyramid_re
         resp = error_views.missing_lti_launch_param_error()
 
         assert pyramid_request.response.status_int == 400
+
+    def test_it_logs_missing_lti_launch_param_error_in_sentry(self, pyramid_request):
+        exc = MissingLTILaunchParamError('test lti launch param error msg')
+
+        error_views = error.ErrorViews(exc, pyramid_request)
+        error_views.missing_lti_launch_param_error()
+
+        assert pyramid_request.raven.captureException.call_count == 1
```

Comment:
Do we want to test that `captureException()` was called with the right args, rather than just how many times it was called? I think in this case the right args are no args, so just `pyramid_request.raven.captureException.assert_called_once_with()`

Reply:
I'll leave it up to you whether to make this change